src/node_process_methods.cc

```diff
@@ -499,6 +500,38 @@ static void LoadEnvFile(const v8::FunctionCallbackInfo<v8::Value>& args) {
   }
 }
 
+static void CveRevert(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  CHECK_EQ(args.Length(), 1);
+  CHECK(args[0]->IsString());
+  Utf8Value cve_string(env->isolate(), args[0]);
+  std::string revert_error;
+
+  Revert(*cve_string, &revert_error);
+  if (revert_error.empty()) {
+    // Revert sets the reversion at the C level but we have to
+    // set the property on the Object as it does not do that
+    Isolate* isolate = env->isolate();
+    Local<Context> context = env->context();
+#define V(code, label, __)                                                     \
+  do {                                                                         \
+    if (strcmp(*cve_string, label) == 0) {                                     \
+      READONLY_PROPERTY(args.This(), REVERT_PREFIX #code, True(isolate));      \
+    }                                                                          \
+  } while (0);
+    SECURITY_REVERSIONS(V)
+#undef V
+  }
+
+  Local<String> revert_error_return =
+      String::NewFromUtf8(env->isolate(),
+                          revert_error.c_str(),
+                          NewStringType::kNormal,
```

Comment:
minor nit: might make sense to intern the string, but definitely not critical. Also, these should likely always be one-byte so maybe `NewFromOneByte` would work to avoid the utf8 transcoding?